From: Jan Blunck <jblunck@suse.de>
Subject: Select FRAME_POINTER only on SYSPROF_TRACER
Date: Wed Sep 24 10:32:16 CEST 2008

The only tracer that requires frame pointers is the sysprof trace. Since this
tracer copies the functionality of oprofile, it isn't required at all.

Signed-off-by: Jan Blunck <jblunck@suse.de>
---
 kernel/trace/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/kernel/trace/Kconfig
===================================================================
--- a/kernel/trace/Kconfig
+++ b/kernel/trace/Kconfig
@@ -18,7 +18,6 @@ config TRACING
 config FTRACE
 	bool "Kernel Function Tracer"
 	depends on HAVE_FTRACE
-	select FRAME_POINTER
 	select TRACING
 	select CONTEXT_SWITCH_TRACER
 	help
@@ -79,6 +78,7 @@ config SYSPROF_TRACER
 	bool "Sysprof Tracer"
 	depends on X86
 	select TRACING
+	select FRAME_POINTER
 	help
 	  This tracer provides the trace needed by the 'Sysprof' userspace
 	  tool.
